Control.FullScreen.js

```diff
@@ -142,8 +144,10 @@
 
 		_screenfull: fullscreenAPI,
 
-		onAdd: function (map) {
-			var className = 'leaflet-control-zoom-fullscreen', container, content = '';
+		onAdd(map) {
```

Comment:
This change of syntax seems legit as leaflet core doest it on master https://github.com/Leaflet/Leaflet/blob/main/src/control/Control.Attribution.js#L39 but it's not the case in https://github.com/Leaflet/Leaflet/blob/main/src/control/Control.Attribution.js#L39 for instance. Any thoughts about this ?

Reply:
I can't quite follow you. It's the same URL twice 🤔

Reply:
wrong paste, my bad ^^

Reply:
Leaflet.heat uses ESLint 1.x, we use 8.x. And the file you linked has not been edited for 6 years. I think we should not use such old code as a reference.